lib/components/mobile/route-viewer.js

```diff
@@ -1,14 +1,14 @@
 import React, { Component } from 'react'
-import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
```

Comment:
Sort imports

Reply:
Sorted in 
092b53f aside from React which I think needs to come first